frontend/src/app/modules/common/tabs/scrollable-tabs.component.ts

```diff
@@ -1,5 +1,11 @@
 import {AfterViewInit, Component, ElementRef, ViewChild} from "@angular/core";
 
+export interface Tab {
+  id:string;
+  name:string;
+  path?:string;
+}
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/32079589a06554b8e496863c812560c7d6677b0e/frontend/src/app/modules/common/tabs/scrollable-tabs.component.ts#L8"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>